resources/rpi/user-data

```diff
@@ -0,0 +1,95 @@
+#cloud-config
+# https://cloudinit.readthedocs.io/
+
+#####################################################################
+# Change instance-specific options here
+#####################################################################
+
+hostname: huntsmanpiXXX
+
+#####################################################################
+# You shouldn't need to change anything below.
+#####################################################################
+
+# If you have set up ssh on github you can pull down your
+# key automatically so that you can log into the unit without
+# a password.
+ssh_import_id:
+  - gh:AstroHuntsman
+
+ntp:
+  enabled: true
+  servers:
+    - time1.google.com
+    - time2.google.com
+    - time3.google.com
+    - time4.google.com
+
+chpasswd:
+  list:
+    - huntsman:huntsman
+
+ssh_pwauth: yes
+
+# New groups to create.
+groups:
+  - huntsman
+  - panoptes
+  - docker
+
+users:
+  - name: huntsman
+    gecos: Huntsman User
+    primary_group: huntsman
+    groups: users, admin, dialout, plugdev, docker, i2c, input, gpio, panoptes
+    sudo: "ALL=(ALL) NOPASSWD:ALL"
+    lock_passwd: false
+    shell: /bin/bash
+
+## Update apt database and upgrade packages on first boot
+package_update: true
+package_upgrade: true
+
+byobu: enable
+
+## Install additional packages on first boot.
+packages:
+  - apt-transport-https
+  - byobu
+  - ca-certificates
+  - git
+  - htop
+  - httpie
+  - jq
+  - neovim
+  - software-properties-common
+  - speedometer
+  - vim-nox
+  - watchdog
+  - sshfs
+
+write_files:
+  # Allow huntsman user to mount via sshfs.
+  - content: |
+      user_allow_other
+    path: /etc/fuse.conf
+    append: true
+
+## Get and run the install script upon first boot.
+runcmd:
+  # Setup hardware watchdog
+  - echo 'watchdog-device = /dev/watchdog' >> /etc/watchdog.conf
+  - echo 'watchdog-timeout = 15' >> /etc/watchdog.conf
+  - echo 'max-load-1 = 24' >> /etc/watchdog.conf
```

Comment:
We might need to think about these a bit. When I tested it I was getting resets when I didn't want to.

Reply:
@wtgee should I remove them for now?